apps-rendering/src/components/Dateline/index.tsx

```diff
@@ -75,18 +78,14 @@ const getDatelineStyles = (format: ArticleFormat): SerializedStyles => {
 	}
 };
 
-const Dateline: FC<Props> = ({ date, format }) =>
+const Dateline: FC<Props> = ({ date, format, edition }) =>
 	pipe(
 		date,
 		map((d) => (
-			<time
-				css={getDatelineStyles(format)}
-				data-date={d}
-				className="date js-date"
```

Comment:
Will removing this class also prevent the datetime being updated client-side for non-blog articles? Should we do one of the following:

1. Leave this class in place until we've migrated those articles over?
2. Migrate those articles over to the new format now as well?

Reply:
Hmm, that's a good point. I'm leaning to option 2 - there isn't significant work to complete the entire migration. I've made the remaining changes in #5766 which could easily be merged here if it's not too much to review at once